bn-models/src/models/trips-api/catches.ts

```diff
@@ -13,6 +13,22 @@ export enum errorType {
   showStopper = 'showStopper',
 }
 
+export enum gearType {
+  trawl = 'trawl',
+  hookAndLine = 'hook & line',
+  fishPot = 'fish pot',
+  longline = 'longline (snap)'
+}
+
+export enum netType {
```

Comment:
same as the above comment, probably belongs in net couch lookup def

Reply:
implementing